### ISSUE TITLE
"Ordinary" Should Be Abbreviated "Ord."

### ISSUE THREAD

**Issue by Fourniture** [musescore/MuseScore#25646]
### Your idea

When the "Ordinary (default)" setting is selected for a sound flag, the text that appears is "ordinary". I suggest that it should always appear as "ord.", because that is the standard. If the user really wants to write "ordinary", he can edit the staff text manually.

### Problem to be solved

Incorrect notation.

### Checklist

- [X] This request follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting issues
- [X] I have verified that this feature request has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar requests

**Comment by FrancRos31** [comment]
I agree with @Fourniture. And, in addition to Ordinary, change Pizzicato in Pizz., too.

**Comment by Jojo-Schmitz** [comment]
I beliebe that pizzicato stems from Muse Sound directly, so can't get changed in MuseScore

**Comment by bkunda** [comment]
While choosing between "ord.", "ordinary", or even "nat." or "norm." is arguably subjective, the broader request here seems to be to give the user the chance to specify how they'd like all sound flag/stave text items to appear (in the manner of presets). This would probably make sense, and is something we could consider tackling in the further future.

**Comment by Fourniture** [comment]
> the broader request here seems to be to give the user the chance to specify how they'd like all sound flag/stave text items to appear (in the manner of presets).

 I am not asking for any presets to specify anything; I am asking that certain sound flags (like "ordinary") appear abbreviated, because there is practically no case in which they should appear in full form. Such a preset feature would likely be entirely unnecessary, due to how rare the full form is; it would make more sense for the user to type it out themselves.
